.nvmrc

```diff
@@ -0,0 +1 @@
+6
```

Comment:
Is this addition intentional?

Reply:
Yup, new CI wants an nvmrc file defined - https://github.com/creative-commoners/silverstripe-advancedworkflow/runs/7194338426?check_suite_focus=true

I don't actually don't know what version it's supposed to be some I'm just guessing :-)